dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -792,12 +793,12 @@ public String getLocaleFacetLabelName(String facetLabelName, String facetCategor
     }
 
     private String getDatasetFieldFacetCategoryName(DatasetFieldType matchedDatasetField) {
-        if (matchedDatasetField.isFacetable()) {
-            if (matchedDatasetField.getParentDatasetFieldType() != null) {
-                return matchedDatasetField.getLocaleTitleWithParent();
+        if (matchedDatasetField.isFacetable() && !matchedDatasetField.isHasParent()) {
+            try {
+                return getStringFromBundle(format(FACETBUNDLE_MASK_VALUE, matchedDatasetField.getName()));
```

Comment:
Now you will get empty string for some datasetfield types - Check `getStringFromBundle` implementation